src/operators.jl

```diff
@@ -459,7 +459,12 @@ end
 # nonlinear function fallbacks for JuMP built-in types
 ###############################################################################
 
-const _OP_HINT = "Are you trying to build a nonlinear problem? Make sure you use @NLconstraint/@NLobjective."
+const _OP_HINT =
+    "Are you trying to build a nonlinear problem? Make sure you use " *
+    "@NLconstraint/@NLobjective. If you are using an `@NL` macro and you " *
+    "encountered this error message, it is because you are attempting to use " *
+    "another unsupported function which calls this method internally."
```

Comment:
Unsupported by what ? Unsupported for applying on a `AffExpr` or unsupported by NL ?

Reply:
Either and both. At the moment, tracing some functions works if they return affine/quadratic expressions.

If you hit this error, then tracing didn't work.